WishList/WishList/Friends/SearchFriendsScreen.swift

```diff
@@ -38,20 +38,33 @@ struct SearchFriendsScreen: View {
         }.searchable(text: $searchText, placement: .navigationBarDrawer(displayMode: .always), prompt: "Search friends")
             .overlay{
                 Button(action: {
-                    performSearch("")
+                    searchFriend()
                 }){Text("LIST") }
             }
         
     }
     
-    private func performSearch(_ term: String) {
-        fullFakeFriends = [
-//        FakeFriend(id: "1", name: "Gian El hombre", urlImage: "", isFriend: false),
-//        FakeFriend(id: "2", name: "Andres el come hombre", urlImage: "", isFriend: false),
-//        FakeFriend(id: "3", name: "Layla", urlImage: "", isFriend: false),
-//        FakeFriend(id: "4", name: "Tu vieja", urlImage: "", isFriend: false)
-        ]
-        showInitialMessage = false
+    private func searchFriend(){
+        guard let url = URL(string:
+            "\(Configuration.baseUrl)/users/search?searchTerm\(searchText)")else{
```

Comment:
`"\(Configuration.baseUrl)/users/search?searchTerm\(searchText)")else{` 
should be 
`"\(Configuration.baseUrl)/users/search?searchTerm=\(searchText)") else {`

Add the = on the url and add spaces on the else